src/sentry/tasks/derive_code_mappings.py

```diff
@@ -168,15 +167,10 @@ def report_project_codemappings(
     set_tag("project.slug", project.slug)
     if code_mappings:
         msg = f"Project {project.slug} would create these code mappings {code_mappings} based on {stacktrace_paths=}"
-        sentry_msg = "derive_code_mappings: NO code mappings would have been created."
     else:
         msg = f"Project {project.slug} would NOT create code mapping based on {stacktrace_paths=}"
-        sentry_msg = "derive_code_mappings: code mappings would have been created."
     existing_code_mappings = RepositoryProjectPathConfig.objects.filter(project=project)
     if existing_code_mappings.exists():
         msg = f"Project {project.slug}: found {existing_code_mappings=} while attempting to create {code_mappings=} for {stacktrace_paths=}"
-        sentry_msg = "derive_code_mappings: code mappings already exist."
-    logger.info(msg)
-    logger.info("Reported to Sentry; Query for 'level:info derive'")
-    # Using a generic message will group all events together
-    sentry_sdk.capture_message(sentry_msg)
+
+    logger.info(msg, extra={"org": project.organization.slug, "project": project.slug})
```

Comment:
should we give these messages a common prefix? makes it easier to search through gcp logs

Reply:
We set the prefix with `logger = logging.getLogger("sentry.tasks.derive_code_mappings")`